app/templates/chat.html

```diff
@@ -1,289 +1,406 @@
-<html>
```

Comment:
Please rebase the file and apply your changes again. (Use the file from the master branch and apply your changes there)